sklearnex/linear_model/logistic_regression.py

```diff
@@ -21,18 +21,6 @@
 from daal4py.sklearn.linear_model.logistic_path import (
     LogisticRegression as LogisticRegression_daal4py,
 )
-from daal4py.sklearn.linear_model.logistic_path import daal4py_fit, daal4py_predict
```

Comment:
Separate PR for this change? Seems unrelated.

Reply:
Sure. This is totally unrelated :)
I actually thought about this. Now I will definitely do a separate PR.
Good catch thank you!

Reply:
Done. Please check #1877